husky_components/matrix/main.js

```diff
@@ -215,21 +217,20 @@ define(function() {
             this.sandbox.dom.append($tr, $tdValue);
 
             //add all link
-            this.sandbox.dom.html(
-                $tdAll,
-                [
+                $tdAll.append(
+                [   '<div class="all">',
```

Comment:
Why does this have to be wrapped in a div? The `display` value is set to`inline-block`, this could also be set on the span. And does it have to `inline-block`? Why can't it be `inline`, the default value of the `span` element?